app/src/main/java/com/poupa/vinylmusicplayer/helper/MusicPlayerRemote.java

```diff
@@ -172,8 +172,11 @@ public static boolean isPlaying() {
     }
 
     public static boolean isPlaying(@NonNull Song song) {
-        if (!isPlaying()) {return false;}
-        return (song.id == getCurrentSong().id);
+        return musicService != null && musicService.isPlaying(song);
+    }
+
+    public static boolean isPlaying(@NonNull IndexedSong song) {
+        return musicService != null && musicService.isPlaying(song);
```

Comment:
I am not sure why this function is necessary, since it is doing the same thing as the one above, and an IndexedSong is also a Song.